bcipy/helpers/raw_data.py

```diff
@@ -0,0 +1,303 @@
+"""Functionality for reading and writing raw signal data."""
+import csv
+from typing import List, TextIO, Tuple
+
+import numpy as np
+import pandas as pd
+
+from bcipy.signal.generator.generator import gen_random_data
+
+TIMESTAMP_COLUMN = 'timestamp'
+
+
+class RawData:
```

Comment:
We spoke about this in person briefly! I think this could reside in acquisition? I don't foresee using this structure outside that module. If you can think of one, we should leave it here.

Reply:
The RawData object is now used in a number of different places including offline_analysis. I think it makes sense to have it at a higher level than acquisition since it's a cross cutting concern. I do think it would make sense to introduce a `core` module for things like this, though. It's not really a helper.